pramen-py/src/pramen_py/metastore/reader.py

```diff
@@ -73,9 +184,13 @@ def get_latest(
         """
 
         until = until or self.info_date
-        logger.info(f"Getting latest partition for {table_name} until {until}")
-
         metastore_table = get_metastore_table(table_name, self.tables)
+        self.validate_partitions_paths(metastore_table)
```

Comment:
Partition validation is only applicable for Parquet, not Delta. Delta can have any structure it wants, we should not look into it and only work with it using spark.read().